diff --git a/package/base-files/files/etc/board.d/99-default_network b/package/base-files/files/etc/board.d/99-default_network
index 49d2a3b..6d19286 100644
--- a/package/base-files/files/etc/board.d/99-default_network
+++ b/package/base-files/files/etc/board.d/99-default_network
@@ -8,8 +8,42 @@ board_config_update
 
 json_is_a network object && exit 0
 
-ucidef_set_interface_lan 'eth0'
-[ -d /sys/class/net/eth1 ] && ucidef_set_interface_wan 'eth1'
+
+ETH_LIST=
+WAN=
+LAN=
+
+for NET_IF in /sys/class/net/eth*; do
+	NET_IF=`basename ${NET_IF}`
+	ETH_LIST=`echo ${ETH_LIST} ${NET_IF}`
+done
+
+echo eth: ${ETH_LIST}
+
+for NET_IF in ${ETH_LIST}; do
+        if [ ! "${WAN}" ]; then
+                WAN=${NET_IF}
+                echo "set wan->${NET_IF}"
+        else
+                LAN=`echo ${LAN} ${NET_IF}`
+                echo "add lan->${NET_IF}"
+        fi
+done
+
+if [ ! "${LAN}" ]; then
+        if [ ! ${WAN} ]; then
+                LAN=eth0
+        else
+                LAN=${WAN}
+                WAN=
+        fi
+fi
+
+echo wan: ${WAN}
+echo lan: ${LAN}
+
+ucidef_set_interface_lan "${LAN}"
+[ ${WAN} ] && ucidef_set_interface_wan "${WAN}"
 
 board_config_flush
 
